packages/victory-core/src/victory-util/selection.ts

```diff
@@ -83,7 +84,7 @@ export function getSVGEventCoordinates(
 export function getDomainCoordinates(
   props: Pick<VictoryCommonProps, "scale" | "horizontal">,
   domain?: DomainPropType,
-): DomainPropType {
+): { x: Tuple<number>; y: Tuple<number> } {
```

Comment:
Not sure if this is the best way to resolve TS errors for `selectedCoords` and `fullCoords` [here](https://github.com/FormidableLabs/victory/pull/2393/files#diff-4db73b037461df1d7256e5f726db803033e0b4a673632a7e895d341df2b1e7a3R57)?

Reply:
I'd say, remove this type (and let it have an implicit return type).  Seems easier.